=== modified file 'libxc-2.0.x/src/Makefile.am'
--- libxc/src/Makefile.am	2013-04-16 17:32:44 +0000
+++ libxc/src/Makefile.am	2013-04-16 17:33:08 +0000
@@ -129,16 +129,23 @@
 xc_funcs.h libxc_funcs.f90 funcs_lda.c funcs_gga.c funcs_hyb_gga.c funcs_mgga.c funcs_hyb_mgga.c funcs_key.c: get_funcs.pl
 	$(srcdir)/get_funcs.pl $(srcdir) $(top_builddir)
 
-$(LIBFUNCMOD):
-	$(MAKE) libxc_funcs.o
-
-libxc.f90 $(XCLIBMODS): libxc_master.F90 libxc_funcs.o
+$(LIBFUNCMOD): libxc_la-libxc_funcs.lo
+$(XCLIBMODS): libxc_la-libxc.lo
+
+libxc_la-libxc.lo: libxc.f90 $(LIBFUNCMOD)
+	$(LIBTOOL)  --tag=FC $(libxc_la_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(FC) $(AM_FCFLAGS) $(FCFLAGS) -c -o libxc_la-libxc.lo $(FCFLAGS_f90) `test -f 'libxc.f90' || echo '$(srcdir)/'`libxc.f90
+
+libxc_la-libxc_s.lo: libxc_s.f90 $(LIBFUNCMOD)
+	$(LIBTOOL)  --tag=FC $(libxc_la_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(FC) $(AM_FCFLAGS) $(FCFLAGS) -c -o libxc_la-libxc_s.lo $(FCFLAGS_f90) `test -f 'libxc_s.f90' || echo '$(srcdir)/'`libxc_s.f90
+
+
+libxc.f90: libxc_master.F90 libxc_funcs.o
 	@FCCPP@ @CPPFLAGS@ $(AM_CPPFLAGS) $(srcdir)/libxc_master.F90 > $(top_builddir)/src/libxc.f90
 	@if [ "@F90_ACCEPTS_LINE_NUMBERS@" = "no" ]; then \
 		grep -v "^#" $(top_builddir)/src/libxc.f90 > $(top_builddir)/src/libxc.f91; \
 		mv -f $(top_builddir)/src/libxc.f91 $(top_builddir)/src/libxc.f90; \
 	fi
-libxc_s.f90 $(XCLIBMODS_S): libxc_master.F90 libxc_funcs.o
+libxc_s.f90: libxc_master.F90 libxc_funcs.o
 	@FCCPP@ -D SINGLE_PRECISION=1 @CPPFLAGS@ $(AM_CPPFLAGS) $(srcdir)/libxc_master.F90 > $(top_builddir)/src/libxc_s.f90
 	@if [ "@F90_ACCEPTS_LINE_NUMBERS@" = "no" ]; then \
 		grep -v "^#" $(top_builddir)/src/libxc_s.f90 > $(top_builddir)/src/libxc_s.f91; \

