=== modified file 'libxc-1.1.0/m4/fc_integer.m4'
--- libxc-1.1.0/m4/fc_integer.m4	2011-07-01 16:01:25 +0000
+++ libxc-1.1.0/m4/fc_integer.m4	2011-08-01 15:20:19 +0000
@@ -46,6 +46,9 @@
   if AC_TRY_EVAL(ac_try); then
     ac_try=""
     ac_fcintegersize=`./intsizetest.x`;
+    if test -z "$ac_fcintegersize" ; then
+      ac_fcintegersize=4
+    fi
   else
     echo "configure: failed program was:" >&AC_FD_CC
     cat intsizetest.f90 >&AC_FD_CC

