=== modified file 'libxc-1.1.0/src/gga_x_pbe.c'
--- libxc-1.1.0/src/gga_x_pbe.c	2011-07-01 16:01:25 +0000
+++ libxc-1.1.0/src/gga_x_pbe.c	2011-07-06 08:15:23 +0000
@@ -61,12 +61,12 @@
   };
 
   static const FLOAT mu[13] = {
-    0.066725*M_PI*M_PI/3.0, /* PBE: mu = beta*pi^2/3, beta = 0.066725 */
-    0.066725*M_PI*M_PI/3.0, /* PBE rev: as PBE */
+    0.2195149727645171, /* PBE: mu = beta*pi^2/3, beta = 0.066725 */
+    0.2195149727645171, /* PBE rev: as PBE */
     10.0/81.0,              /* PBE sol */
     0.23214,                /* xPBE */
     0.046*M_PI*M_PI/3.0,    /* PBE_JSJR */
-    0.066725*M_PI*M_PI/3.0, /* PBEK1_VDW: as PBE */
+    0.2195149727645171, /* PBEK1_VDW: as PBE */
     10.0/81.0,              /* RGE2      */
     0.260,                  /* APBE (X)  */
     0.23889,                /* APBE (K)  */

=== modified file 'libxc-1.1.0/src/util.h'
--- libxc-1.1.0/src/util.h	2011-07-01 16:01:25 +0000
+++ libxc-1.1.0/src/util.h	2011-07-04 14:21:20 +0000
@@ -70,9 +70,9 @@
 #define D3FZETA(x)     (-(8.0/27.0)/FZETAFACTOR)* \
   (ABS(x)==1.0 ? (FLT_MAX) : (pow(1.0 + (x), -5.0/3.0) - pow(1.0 - (x), -5.0/3.0)))
 
-#define MIN_DENS             5.0e-13
-#define MIN_GRAD             5.0e-13
-#define MIN_TAU              5.0e-13
+#define MIN_DENS             1.0e-20
+#define MIN_GRAD             1.0e-20
+#define MIN_TAU              1.0e-20
 
 #include "xc.h"
 

=== modified file 'libxc-1.1.0/src/Makefile.am'
--- libxc-1.1.0/src/Makefile.am	2011-07-01 16:01:25 +0000
+++ libxc-1.1.0/src/Makefile.am	2011-07-08 08:13:08 +0000
@@ -128,6 +128,13 @@
 $(LIBFUNCMOD):
 	$(MAKE) libxc_funcs.o
 
+libxc_la-libxc.lo: libxc_la-libxc_funcs.lo libxc.f90
+	$(LIBTOOL)  --tag=FC $(libxc_la_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(FC) $(AM_FCFLAGS) $(FCFLAGS) -c -o libxc_la-libxc.lo $(FCFLAGS_f90) `test -f 'libxc.f90' || echo '$(srcdir)/'`libxc.f90
+
+libxc_la-libxc_s.lo: libxc_la-libxc_funcs.lo libxc_s.f90
+	$(LIBTOOL)  --tag=FC $(libxc_la_LIBTOOLFLAGS) $(LIBTOOLFLAGS) --mode=compile $(FC) $(AM_FCFLAGS) $(FCFLAGS) -c -o libxc_la-libxc_s.lo $(FCFLAGS_f90) `test -f 'libxc_s.f90' || echo '$(srcdir)/'`libxc_s.f90
+
+
 libxc.f90 $(XCLIBMODS): libxc_master.F90
 	@FCCPP@ @CPPFLAGS@ $(AM_CPPFLAGS) $(srcdir)/libxc_master.F90 > $(top_builddir)/src/libxc.f90
 	@if [ "@F90_ACCEPTS_LINE_NUMBERS@" = "no" ]; then \

